src/app/pages/manage-organisation/manage-organisation-registration-step-3/manage-organisation-registration-step-3.component.html

```diff
@@ -18,16 +33,22 @@ <h1 class="govuk-heading-xl page-title">
         {{ i?.address?.postalCode }}<br *ngIf="i?.address?.postalCode" />
         {{ i?.address?.countryName }}<br *ngIf="i?.address?.countryName" />
       </p>
-      <div class="govuk-form-group" id="country-code-form-group">
+      <div class="govuk-form-group" id="country-code-form-group"
+      [class.govuk-form-group--error]="isValid == true"
```

Comment:
Shall we change the variable name to "isInvalid"  (because, if its invalid we show the error)